cfgov/v1/routing.py

```diff
@@ -54,7 +54,7 @@ def get_protected_url(context, page):
     if page:
```

Comment:
I'm wondering if this entire `get_protected_url` method can be removed. Maybe better left for a future PR so as not to make this one any larger than it already is. But is there any more use case for this?